### PR TITLE
Add Handlers for stock transfers

### PR DESCRIPTION
**Pull request by allisonlarson**
Added handler for when stock transfer shipments are created in wombat,
and handler for when a stock transfer has been shipped.

* Added add_stock_transfer_handler
* Added update_stock_transfer_handler
* Specs

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'spree', github: 'bonobos/spree', branch: '2-2-dev'
+gem 'spree', github: 'bonobos/spree', branch: 'stock_transfer_mgmt'
```
Comment (allisonlarson) [inline Gemfile]: Note to change back! (When merged to spree/this is ready to merge)

File: lib/spree/wombat/handler/add_stock_transfer_handler.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            return response "Could not find #{e.message} for transfer shipment: #{number}", 500
+          end
+
+          response "Successfully made stock transfer #{number}", 200
+        end
+
+        private
+
+        def get_location(code)
+          code = 'QLAYR' if code == 'DVN'
```
Comment (athal7) [inline lib/spree/wombat/handler/add_stock_transfer_handler.rb]: i don't think this is for everyone

File: Gemfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'spree', github: 'bonobos/spree', branch: '2-2-dev'
+gem 'spree', github: 'bonobos/spree', branch: 'stock_transfers_mgmt'
```
Comment (allisonlarson) [inline Gemfile]: Note to change back when https://github.com/bonobos/spree/pull/334 is merged

**Comment by athal7** [comment]
lgtm

**Comment by athal7** [comment]
:shipit:

**Pull request merged**